src/GridView/index.js

```diff
@@ -11,20 +12,31 @@ const CLASS_ROOT = 'pcui-gridview';
  * @classdesc Represents a container that shows a flexible wrappable
  * list of items that looks like a grid. Contains GridViewItem's.
  * @property {GridViewItem[]} selected Gets the selected grid view items.
+ * @property {boolean} vertical If true the gridview layout will be vertical.
+ * @property {boolean} multiSelect=true If true, the layout will allow for multiple options to be selected.
+ * @property {boolean} allowDeselect=true If true and multiSelect is set to false, the layout will allow options to be deselected.
  */
 class GridView extends Container {
     /**
      * Creates new GridView.
      *
      * @param {object} [args] - The arguments
      * @param {Function} [args.filterFn] - A filter function to filter gridview items with signature (GridViewItem) => boolean.
+     * @param {boolean} [args.vertical] - Whether or not the layout will be vertically aligned
+     * @param {boolean} [args.multiSelect] - Whether or not the layout will allow for multiple items to be selected at once.
+     * @param {boolean} [args.allowDeselect] - Whether or not the layout will allow for options to be deselected.
      */
     constructor(args) {
         if (!args) args = {};
 
         super(args);
 
-        this.class.add(CLASS_ROOT);
+        this._vertical = args.vertical;
```

Comment:
This value will usually be undefined so it might be worth setting this as follows to make the _booleanness_ of it clear:
```javascript
this._vertical = !!args.vertical;
```